### PR TITLE
Leads page list component is complete

### PR DESCRIPTION
**Pull request by Saidul-M-Khan**

### CODE REVIEW

**Comment by Saidul-M-Khan** [comment]
Leads page's table component is complete

File: components/ui/LeadsList/LeadsList.tsx [hunk 1]
```diff
@@ -0,0 +1,121 @@
+import { LEADS_DATA } from '@/utils/constants/leadslist-constant';
+import Image from 'next/image';
+import plus from '/public/leadslist-icons/add-circle.png';
+import phone from '/public/leadslist-icons/call.png';
+import clock from '/public/leadslist-icons/clock.png';
+import more from '/public/leadslist-icons/more_vert.png';
+
+// Define a function to get the appropriate color based on status
+const getStatusColor = (status: string) => {
```
Comment (tajnur007) [inline components/ui/LeadsList/LeadsList.tsx:9]: I will suggest you use this method as a utility function, so you have to move it to `utils -> helpers -> common-helpers.ts` file.

File: components/ui/LeadsList/LeadsList.tsx [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+                  </div>
+                </button>
+              </div>
+            </div>
+          </div>
+          <div className='overflow-x-auto overflow-y-auto'>
+            <table className="w-full  whitespace-nowrap [font-family:'Metropolis-Bold',Helvetica] font-medium text-[14px] leading-[normal]">
+              <tbody className='w-full'>
+                {LEADS_DATA.map((item, index) => (
+                  <tr
```
Comment (tajnur007) [inline components/ui/LeadsList/LeadsList.tsx:62]: Make a separate component for this `<tr>`. We can name it as `<LeadRow>`, that will help use to keep our code more understandable and clean.

File: utils/constants/leadslist-constant.js [hunk 3]
```diff
@@ -0,0 +1,101 @@
+export const LEADS_DATA = [
+  {
+    id: 1,
+    title: 'Social Media Marketing',
+    date: 'Tue 21 Nov, 2023 11:34 AM',
+    assignedByName: 'Saidul M Khan',
+    assignedByNumber: '+88-01623364856',
+    assignedToName: 'Rezaul Mursalin Khan',
+    status: 'Hot',
```
Comment (tajnur007) [inline utils/constants/leadslist-constant.js:9]: Keep status into `common-constant.ts` file, we can keep it as: 
```javascript 
LEAD_STATUS = {
  Hot: 'Hot',
  InProgress: 'In Progress',
......................................
.....................................
......................................
};
```
Then we can use it here as:
```javascript
status: LEAD_STATUS.Hot,
```

**Comment by Saidul-M-Khan** [comment]
Updated as per instructions
screencapture-localhost-3000-leads-2024-01-06-00_17_47

**Comment by Saidul-M-Khan** [comment]
Facing these issue

screencapture-localhost-3000-leads-2024-01-06-00_49_47
Screenshot_177

**Comment by Saidul-M-Khan** [comment]
All bugs are fixed!
screencapture-localhost-3001-leads-2024-01-06-17_19_48

**Pull request merged**